aip/general/0127.md

```diff
@@ -116,6 +116,9 @@ rpc CreateBook(CreateBookRequest) returns (Book) {
       post: "/v1/books"
       body: "book"
     }
+    additional_bindings: {
+      get: "/v1/books:Create"
```

Comment:
regardless of whether we allow GET / POST mappings, we definitely shouldn't add this example - this is a Create RPC, and creates *must* use POST along with map to a resource pattern: https://google.aip.dev/133#guidance